pkgs/applications/science/math/singular/default.nix

```diff
@@ -85,23 +132,62 @@ stdenv.mkDerivation rec {
     # do nothing
   '';
 
+  doCheck = true; # very basic checks, does not test any libraries
+
   installPhase = ''
     mkdir -p "$out"
     cp -r Singular/LIB "$out/lib"
     make install
-
+  '' + lib.optionalString enableDocs ''
+    # Sage uses singular.hlp (which is not in the tarball)
+    mkdir -p $out/share/info
+    cp doc/singular.hlp $out/share/info
+  '' + ''
     # Make sure patchelf picks up the right libraries
     rm -rf libpolys factory resources omalloc Singular
   '';
 
+  # singular tests are a bit complicated, see
+  # https://github.com/Singular/Singular/tree/spielwiese/Tst
+  # https://www.singular.uni-kl.de/forum/viewtopic.php&t=2773
```

Comment:
This is a 404, the link should be https://www.singular.uni-kl.de/forum/viewtopic.php?f=10&t=2773. I wrote this comment myself in #45439, not sure why the `f=10` is missing.

Reply:
I guess the forum software was upgraded in the meantime. I actually visited the URL and got the 404, should have updated it myself :)

Reply:
Maybe I just removed that parameter for some reason. Thanks for fixing it in #122592 :)